system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -0,0 +1,11 @@
+const React = require("react");
+const {connect} = require("react-redux");
+
+const TopSites = props => (<section id="top-sites-container">
```

Comment:
nit: unused css id...do we need a top-sites-container or is this here for future top sites css?